x-pack/legacy/plugins/ml/public/overview/components/anomaly_detection_panel/anomaly_detection_panel.tsx

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, Fragment, useState, useEffect } from 'react';
+import {
+  EuiButton,
+  EuiButtonEmpty,
+  EuiCallOut,
+  EuiEmptyPrompt,
+  EuiLoadingSpinner,
+  EuiPanel,
+  EuiSpacer,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { toastNotifications } from 'ui/notify';
+import { AnomalyDetectionTable } from './table';
+import { ml } from '../../../services/ml_api_service';
+import { mlResultsService } from '../../../services/results_service';
+import { getGroupsFromJobs, getStatsBarData, getJobsWithTimerange } from './utils';
+
+export interface Groups {
+  [key: string]: Group;
+}
+
+export interface Group {
+  id: string;
+  jobIds: string[];
+  docs_processed: number;
+  earliest_timestamp: number;
+  latest_timestamp: number;
+  max_anomaly_score: number | null;
+}
+
+export interface Job {
```

Comment:
That's not quite the type I need since I'm getting jobs from `ml.jobsSummary` but I went ahead and added a `MlSummaryJob` and `MlSummaryJobs` as we'll need those types when we convert all the job management code to typescript.